### PR TITLE
Update download.md

### PR DESCRIPTION
**Pull request by rohitg00**
Hello,

After the launch of [Kubernetes v1.25 release](https://kubernetes.io/blog/2022/08/23/kubernetes-v1-25-release/), I noticed there is a default change in the pull release of container images. Hence, This needs to be visible on the Kubernetes release website page. This works for all images, not just v1.25.
- this needs to fix as it will help automate further for new projects.
- screenshot attached.

Screenshot 2022-08-25 at 2 01 21 PM

### CODE REVIEW

File: content/en/releases/download.md [hunk 1]
```diff
@@ -13,22 +13,22 @@ for multiple operating systems as well as hardware architectures.
 ## Container Images
 
 All Kubernetes container images are deployed to the
-[k8s.gcr.io](https://console.cloud.google.com/gcr/images/k8s-artifacts-prod/GLOBAL)
+[registry.k8s.io](https://console.cloud.google.com/gcr/images/k8s-artifacts-prod/GLOBAL)
 container registry.
 
-{{< feature-state for_k8s_version="v1.24" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.25" state="alpha" >}}
```
Comment (sftim) [inline content/en/releases/download.md:19]: This is an alpha feature from v1.24; the original text was correct.

```suggestion
{{< feature-state for_k8s_version="v1.24" state="alpha" >}}
```

**Review by sftim — COMMENTED** [review]
Makes sense given https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.25.md#moved-container-registry-service-from-k8sgcrio-to-registryk8sio

However, there is a mistake in the PR (see inline feedback)

File: content/en/releases/download.md [hunk 2]
```diff
@@ -13,22 +13,22 @@ for multiple operating systems as well as hardware architectures.
 ## Container Images
 
 All Kubernetes container images are deployed to the
-[k8s.gcr.io](https://console.cloud.google.com/gcr/images/k8s-artifacts-prod/GLOBAL)
+[registry.k8s.io](https://console.cloud.google.com/gcr/images/k8s-artifacts-prod/GLOBAL)
 container registry.
```
Comment (sftim) [inline content/en/releases/download.md:17]:
```suggestion
`registry.k8s.io` container image registry.
```

**Review by sftim — COMMENTED** [review]
Changes LGTM

File: content/en/blog/_posts/2020-05-27-An-Introduction-to-the-K8s-Infrastructure-Working-Group.md [hunk 3]
```diff
@@ -55,7 +55,7 @@ The team has made progress in the last few months that is well worth celebrating
 
 - The K8s-Infrastructure Working Group released an automated billing report that they start every meeting off by reviewing as a group. 
 - DNS for k8s.io and kubernetes.io are also fully [community-owned](https://groups.google.com/g/kubernetes-dev/c/LZTYJorGh7c/m/u-ydk-yNEgAJ), with community members able to [file issues](https://github.com/kubernetes/k8s.io/issues/new?assignees=&labels=wg%2Fk8s-infra&template=dns-request.md&title=DNS+REQUEST%3A+%3Cyour-dns-record%3E) to manage records.
-- The container registry [k8s.gcr.io](https://github.com/kubernetes/k8s.io/tree/main/k8s.gcr.io) is also fully community-owned and available for all Kubernetes subprojects to use. 
+- The container registry [registry.k8s.io](https://github.com/kubernetes/k8s.io/wiki/New-Registry-url-for-Kubernetes-\(registry.k8s.io\)) is also fully community-owned and available for all Kubernetes subprojects to use.
```
Comment (sftim) [inline content/en/blog/_posts/2020-05-27-An-Introduction-to-the-K8s-Infrastructure-Working-Group.md:58]: Hmm. Should we have edited this blog article? We usually don't, and we're changing the meaning of someone else's article.

Comment (rohitg00) [reply content/en/blog/_posts/2020-05-27-An-Introduction-to-the-K8s-Infrastructure-Working-Group.md:58]: Tricky! It's needed at the moment, If we're maintaining default changes.

Comment (reylejano) [reply content/en/blog/_posts/2020-05-27-An-Introduction-to-the-K8s-Infrastructure-Working-Group.md:58]: I'm open to reverting the change to the blog and adding a note that the container registry is updated to registry.k8s.io

Comment (reylejano) [reply content/en/blog/_posts/2020-05-27-An-Introduction-to-the-K8s-Infrastructure-Working-Group.md:58]: Follow-up PR https://github.com/kubernetes/website/pull/36270

**Pull request merged**